src/components/LayoutSingleColumn/LayoutSingleColumn.js

```diff
@@ -11,7 +11,7 @@ import css from './LayoutSingleColumn.module.css';
 
 const prepareChildren = children => {
   const childrenCount = React.Children.count(children);
-  if (!(childrenCount === 3 || childrenCount === 4)) {
+  if (!(childrenCount === 2 || childrenCount === 3)) {
```

Comment:
I guess this has been a copy paster error from LayoutSideNavigation? Before the LoadableComponentErrorBoundaryPage we didn't have any page that would use LayoutSingleColumn without footer so that's why this has maybe not been noticed.

Reply:
Change also that ”Menu” on next line :)